docs/index.rst

```diff
@@ -1,106 +1,79 @@
-
-  Nearly complete support for the stable CWL v1.0 specification,
-  allowing it to execute CWL workflows.
-
-* Cross platform support
-
-  Develop and test on your laptop then deploy on any of the following:
-
-  - Commercial clouds:
-    - `Amazon Web Services`_ (including the `spot market`_)
-    - `Microsoft Azure`_
-    - `Google Compute Engine`_
-  - Private clouds:
-    - `OpenStack`_
-  - High Performance Computing Environments:
-    - `GridEngine`_
-    - `Apache Mesos`_
-    - `Parasol`_
-    - Individual multi-core machines
-
-* A small API
-
-  Easily mastered, the Python user API for defining and running workflows is built upon one core class.
-
-* Complete file and stream management:
-
-  Temporary and persistent file management that abstracts the details of the
-  underlying file system, providing a uniform interface regardless of
-  environment. Supports both atomic file transfer and streaming interfaces, and
-  provides encryption of user data.
-
-* Scalability:
-
-  Toil can easily handle workflows concurrently using hundreds of nodes and
-  thousands of cores.
-
-* Robustness:
-
-  Toil workflows support arbitrary worker and leader failure, with strong
-  check-pointing that always allows resumption.
-
-* Efficiency:
+Toil is an open-source pure-Python workflow engine that lets people write better
+pipelines. You can:
 
-  Caching, fine grained, per task, resource requirement specifications, and
-  support for the AWS spot market mean workflows can be executed with little
-  waste.
+* Write your workflows in `Common Workflow Language`_ (CWL),
+* Run workflows on your laptop or on huge commercial clouds such as
+  `Amazon Web Services`_ (including the `spot market`_), `Microsoft Azure`_,
+  `OpenStack`_, and `Google Compute Engine`_,
+* Take advantage of high-performance computing environments with batch systems
+  like `GridEngine`_, `Apache Mesos`_, and `Parasol`_,
+* Run workflows concurrently at scale using hundreds of nodes and thousands of cores,
+* Execute workflows efficiently with caching and resource requirement specifications, and
+* Easily link databases and services
 
-* Declarative and dynamic workflow creation:
-
-  Workflows can be declared statically, but new jobs can be added dynamically
-  during execution within any existing job, allowing arbitrarily complex
-  workflow graphs with millions of jobs within them.
-
-* Support for databases and services:
-
-  For example, Apache Spark clusters can be created quickly and easily
-  integrated within a toil workflow as a service, with precisely defined time
-  start and end times that fits with the flow of other jobs in the workflow.
-
-* Open Source: An Apache license allows unrestricted use, incorporation and modification.
+Toil is, admittedly, not quite as good as sliced bread, but it's as close to it
+as you're gonna get. Click `here`_ to learn more about Toil and what it can do,
+or jump in and skip to :ref:`Installation`. (You can also join us on `GitHub`_
```

Comment:
It'd be great if we could get a short code example here, ideally something that does non-trivial work that benefits from parallelism. When you come in next we might want to sit down and think of a good example to showcase Toil.